include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -490,33 +496,36 @@ struct permutation_view_simple<_R, _M, ::std::enable_if_t<oneapi::dpl::__interna
     }
 
     auto
-    base() const -> decltype(__r)
+    base() const
```

Comment:
The behavior might be different with your change. Let's make sure that the current one is what we want.

Reply:
why different?

Reply:
As we checked In given case there is difference.